File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.py
a = int(input())

for _ in range(a):
  floor, room, guest = map(int, input().split())

  room_number = 1
  while(True):
    if (floor >= guest):
      break
    else:
      guest -= floor

    room_number += 1;

  result = guest * 100 + room_number

  print(result)

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.c
#include <stdio.h>

int main(void) {  
   int T, floor, room, guest, selected_room, result;
   scanf("%d", &T);
 
   for(int j = 0; j < T; j++){
      scanf("%d %d %d", &floor, &room, &guest);
      selected_room = 1;
      while(1){
         if (floor >= guest){
            break;
         }
         else{
            guest -= floor;
         }
         selected_room += 1;
      }

      result = 100 * guest + selected_room;

      printf("%d\n", result);
   }

   return 0;
}